azure-pipelines.yml

```diff
@@ -180,6 +180,46 @@ jobs:
         BUILD_ID: $(Build.BuildId)
       workingDirectory: '$(System.DefaultWorkingDirectory)'
 
+    # TODO: There is no way to run steps in parallel in Azure pipelines but ideally this step would run in parallel
+    # TODO: with the above testing step to reduce overall runtime
+    - script: |
+        set -e
+        export PATH=$PATH:$(go env GOPATH)/bin:$(go env GOPATH)/kubebuilder/bin
+        export KUBEBUILDER_ASSETS=$(go env GOPATH)/kubebuilder/bin
+        make test-webhooks-only-mode
+      displayName: Run webhooks-only mode tests
+      condition: or(eq(variables['check_changes.SOURCE_CODE_CHANGED'], 'true'), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+      continueOnError: 'false'
+      env:
+        GO111MODULE: on
```

Comment:
I feel great sadness for the way this job is now. (That's not your fault nor did I expect you to fix it here, it just means I wish it were better!).